### ISSUE TITLE
NC 14 UI not completed/usable in IE11

### ISSUE THREAD

**Issue by dizdi** [nextcloud/server#11141]
### Steps to reproduce
1. Upgarde to NC 14
2. Open NC in IE 11

### Expected behaviour
UI should look like in Chrome

### Actual behaviour
No bulletmenu, incomplete UI 

### Server configuration

Demo is also affected. See: https://demo14.nextcloud.bayton.org
![nextcloud](https://user-images.githubusercontent.com/23087749/45298708-a3aa9a00-b509-11e8-83c3-d8072430970c.PNG)

**Operating system**:

**Web server:**

**Database:**

**PHP version:**

**Nextcloud version:** (see Nextcloud admin page)

**Updated from an older Nextcloud/ownCloud or fresh install:**

**Where did you install Nextcloud from:**

**Signing status:**
<details>
<summary>Signing status</summary>

```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or 

Insert your config.php content here. 
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** yes/no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

#### LDAP configuration (delete this part if not used)
<details>
<summary>LDAP config</summary>

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your Nextcloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```
</details>

### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
<details>
<summary>Web server error log</summary>

```
Insert your webserver log here
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
Insert your Nextcloud log here
```
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```
</details>

**Comment by jospoortvliet** [comment]
Could you include the browser javascript log, some more screenshots of exactly what is wrong etc so somebody can try to fix it?

Thanks!

**Comment by dizdi** [comment]
@jospoortvliet It's clearly visible what is wrong if opened in IE11. Anybode who's able to fix it, will have to use IE11 anyway. I can't see any benefit in adding more screenshots.

**Comment by Kaeltis** [comment]
Same issue here, one of our customers is using IE11 and since 14 many UI elements are missing, from the console it looks like much of the CSS is broken, here are two examples:

File List:
```
HTML1300: Navigation wurde ausgeführt.
files
DOM7011: Der Code auf dieser Seite hat die Zwischenspeicherung für das Vor- und Zurücknavigieren deaktiviert. Weitere Informationen dazu finden Sie unter: http://go.microsoft.com/fwlink/?LinkID=291337
files
HTML1500: Die Markierung kann nicht selbstschließend sein. Verwenden Sie eine explizite schließende Markierung.
files (106,21)
HTML1500: Die Markierung kann nicht selbstschließend sein. Verwenden Sie eine explizite schließende Markierung.
files (118,21)
HTML1500: Die Markierung kann nicht selbstschließend sein. Verwenden Sie eine explizite schließende Markierung.
files (130,21)
HTML1500: Die Markierung kann nicht selbstschließend sein. Verwenden Sie eine explizite schließende Markierung.
files (142,21)
HTML1508: Endmarkierung ohne Entsprechung.
files (226,50)
JQMIGRATE: Migrate is installed, version 1.4.0
SCRIPT1002: Syntaxfehler
notifications.js (7,98096)
CSS transform warning: variable "--color-text-details" is undefined
CSS transform warning: variable "--icon-confirm-fade-000" is undefined
CSS transform warning: variable "--color-checkbox-radio-border" is undefined
CSS transform warning: variable "--color-checkbox-radio-border" is undefined
CSS transform warning: variable "--color-checkbox-radio-disabled" is undefined
CSS transform warning: variable "--color-checkbox-radio-disabled" is undefined
CSS transform warning: variable "--icon-add-000" is undefined
CSS transform warning: variable "--icon-add-fff" is undefined
CSS transform warning: variable "--icon-address-000" is undefined
CSS transform warning: variable "--icon-address-fff" is undefined
CSS transform warning: variable "--icon-audio-000" is undefined
CSS transform warning: variable "--icon-audio-fff" is undefined
CSS transform warning: variable "--icon-audio-off-000" is undefined
CSS transform warning: variable "--icon-audio-off-fff" is undefined
CSS transform warning: variable "--icon-caret-fff" is undefined
CSS transform warning: variable "--icon-caret-000" is undefined
CSS transform warning: variable "--icon-checkmark-000" is undefined
CSS transform warning: variable "--icon-checkmark-fff" is undefined
CSS transform warning: variable "--icon-checkmark-46ba61" is undefined
CSS transform warning: variable "--icon-clippy-000" is undefined
CSS transform warning: variable "--icon-clippy-fff" is undefined
CSS transform warning: variable "--icon-close-000" is undefined
CSS transform warning: variable "--icon-close-fff" is undefined
CSS transform warning: variable "--icon-comment-000" is undefined
CSS transform warning: variable "--icon-comment-fff" is undefined
CSS transform warning: variable "--icon-confirm-000" is undefined
CSS transform warning: variable "--icon-confirm-fff" is undefined
CSS transform warning: variable "--icon-download-000" is undefined
CSS transform warning: variable "--icon-download-fff" is undefined
CSS transform warning: variable "--icon-confirm-fade-000" is undefined
CSS transform warning: variable "--icon-delete-000" is undefined
CSS transform warning: variable "--icon-delete-000" is undefined
CSS transform warning: variable "--icon-delete-e9322d" is undefined
CSS transform warning: variable "--icon-delete-fff" is undefined
CSS transform warning: variable "--icon-delete-fff" is undefined
CSS transform warning: variable "--icon-delete-fff" is undefined
CSS transform warning: variable "--icon-delete-e9322d" is undefined
CSS transform warning: variable "--icon-details-000" is undefined
CSS transform warning: variable "--icon-details-fff" is undefined
CSS transform warning: variable "--icon-edit-000" is undefined
CSS transform warning: variable "--icon-edit-fff" is undefined
CSS transform warning: variable "--icon-error-000" is undefined
CSS transform warning: variable "--icon-error-fff" is undefined
CSS transform warning: variable "--icon-error-e9322d" is undefined
CSS transform warning: variable "--icon-external-000" is undefined
CSS transform warning: variable "--icon-external-fff" is undefined
CSS transform warning: variable "--icon-fullscreen-000" is undefined
CSS transform warning: variable "--icon-fullscreen-fff" is undefined
CSS transform warning: variable "--icon-history-000" is undefined
CSS transform warning: variable "--icon-history-fff" is undefined
CSS transform warning: variable "--icon-info-000" is undefined
CSS transform warning: variable "--icon-info-fff" is undefined
CSS transform warning: variable "--icon-logout-000" is undefined
CSS transform warning: variable "--icon-logout-fff" is undefined
CSS transform warning: variable "--icon-mail-000" is undefined
CSS transform warning: variable "--icon-mail-fff" is undefined
CSS transform warning: variable "--icon-menu-000" is undefined
CSS transform warning: variable "--icon-menu-fff" is undefined
CSS transform warning: variable "--icon-more-000" is undefined
CSS transform warning: variable "--icon-more-fff" is undefined
CSS transform warning: variable "--icon-password-000" is undefined
CSS transform warning: variable "--icon-password-fff" is undefined
CSS transform warning: variable "--icon-pause-000" is undefined
CSS transform warning: variable "--icon-pause-fff" is undefined
CSS transform warning: variable "--icon-play-000" is undefined
CSS transform warning: variable "--icon-play-fff" is undefined
CSS transform warning: variable "--icon-play-add-000" is undefined
CSS transform warning: variable "--icon-play-add-fff" is undefined
CSS transform warning: variable "--icon-play-next-000" is undefined
CSS transform warning: variable "--icon-play-next-fff" is undefined
CSS transform warning: variable "--icon-play-previous-000" is undefined
CSS transform warning: variable "--icon-play-previous-fff" is undefined
CSS transform warning: variable "--icon-public-000" is undefined
CSS transform warning: variable "--icon-public-fff" is undefined
CSS transform warning: variable "--icon-quota-000" is undefined
CSS transform warning: variable "--icon-quota-fff" is undefined
CSS transform warning: variable "--icon-rename-000" is undefined
CSS transform warning: variable "--icon-rename-fff" is undefined
CSS transform warning: variable "--icon-screen-000" is undefined
CSS transform warning: variable "--icon-screen-fff" is undefined
CSS transform warning: variable "--icon-screen-off-000" is undefined
CSS transform warning: variable "--icon-screen-off-fff" is undefined
CSS transform warning: variable "--icon-search-000" is undefined
CSS transform warning: variable "--icon-search-fff" is undefined
CSS transform warning: variable "--icon-settings-000" is undefined
CSS transform warning: variable "--icon-settings-dark-000" is undefined
CSS transform warning: variable "--icon-settings-dark-fff" is undefined
CSS transform warning: variable "--icon-share-000" is undefined
CSS transform warning: variable "--icon-share-fff" is undefined
CSS transform warning: variable "--icon-share-fff" is undefined
CSS transform warning: variable "--icon-sound-000" is undefined
CSS transform warning: variable "--icon-sound-fff" is undefined
CSS transform warning: variable "--icon-sound-off-000" is undefined
CSS transform warning: variable "--icon-sound-off-fff" is undefined
CSS transform warning: variable "--icon-star-dark-000" is undefined
CSS transform warning: variable "--icon-star-000" is undefined
CSS transform warning: variable "--icon-star-fff" is undefined
CSS transform warning: variable "--icon-star-000" is undefined
CSS transform warning: variable "--icon-star-dark-FC0" is undefined
CSS transform warning: variable "--icon-star-000" is undefined
CSS transform warning: variable "--icon-star-dark-FC0" is undefined
CSS transform warning: variable "--icon-tag-000" is undefined
CSS transform warning: variable "--icon-tag-fff" is undefined
CSS transform warning: variable "--icon-timezone-000" is undefined
CSS transform warning: variable "--icon-timezone-fff" is undefined
CSS transform warning: variable "--icon-toggle-000" is undefined
CSS transform warning: variable "--icon-toggle-fff" is undefined
CSS transform warning: variable "--icon-toggle-background-000" is undefined
CSS transform warning: variable "--icon-toggle-background-fff" is undefined
CSS transform warning: variable "--icon-toggle-pictures-000" is undefined
CSS transform warning: variable "--icon-toggle-pictures-fff" is undefined
CSS transform warning: variable "--icon-triangle-e-000" is undefined
CSS transform warning: variable "--icon-triangle-e-fff" is undefined
CSS transform warning: variable "--icon-triangle-n-000" is undefined
CSS transform warning: variable "--icon-triangle-n-fff" is undefined
CSS transform warning: variable "--icon-triangle-s-000" is undefined
CSS transform warning: variable "--icon-triangle-s-fff" is undefined
CSS transform warning: variable "--icon-upload-000" is undefined
CSS transform warning: variable "--icon-upload-fff" is undefined
CSS transform warning: variable "--icon-user-000" is undefined
CSS transform warning: variable "--icon-user-fff" is undefined
CSS transform warning: variable "--icon-group-000" is undefined
CSS transform warning: variable "--icon-group-fff" is undefined
CSS transform warning: variable "--icon-video-000" is undefined
CSS transform warning: variable "--icon-video-fff" is undefined
CSS transform warning: variable "--icon-video-off-000" is undefined
CSS transform warning: variable "--icon-video-off-fff" is undefined
CSS transform warning: variable "--icon-video-switch-000" is undefined
CSS transform warning: variable "--icon-video-switch-fff" is undefined
CSS transform warning: variable "--icon-view-close-000" is undefined
CSS transform warning: variable "--icon-view-close-fff" is undefined
CSS transform warning: variable "--icon-view-download-000" is undefined
CSS transform warning: variable "--icon-view-download-fff" is undefined
CSS transform warning: variable "--icon-arrow-right-000" is undefined
CSS transform warning: variable "--icon-arrow-right-fff" is undefined
CSS transform warning: variable "--icon-view-pause-000" is undefined
CSS transform warning: variable "--icon-view-pause-fff" is undefined
CSS transform warning: variable "--icon-view-play-000" is undefined
CSS transform warning: variable "--icon-view-play-fff" is undefined
CSS transform warning: variable "--icon-arrow-left-000" is undefined
CSS transform warning: variable "--icon-arrow-left-fff" is undefined
CSS transform warning: variable "--icon-disabled-user-000" is undefined
CSS transform warning: variable "--icon-disabled-user-fff" is undefined
CSS transform warning: variable "--icon-disabled-users-000" is undefined
CSS transform warning: variable "--icon-disabled-users-fff" is undefined
CSS transform warning: variable "--icon-user-admin-000" is undefined
CSS transform warning: variable "--icon-user-admin-fff" is undefined
CSS transform warning: variable "--icon-calendar-fff" is undefined
CSS transform warning: variable "--icon-calendar-000" is undefined
CSS transform warning: variable "--icon-contacts-fff" is undefined
CSS transform warning: variable "--icon-contacts-000" is undefined
CSS transform warning: variable "--icon-files-fff" is undefined
CSS transform warning: variable "--icon-files-000" is undefined
CSS transform warning: variable "--icon-text-000" is undefined
CSS transform warning: variable "--icon-folder-000" is undefined
CSS transform warning: variable "--icon-folder-fff" is undefined
CSS transform warning: variable "--icon-folder-000" is undefined
CSS transform warning: variable "--icon-folder-drag-accept-000" is undefined
CSS transform warning: variable "--icon-home-000" is undefined
CSS transform warning: variable "--icon-home-fff" is undefined
CSS transform warning: variable "--icon-link-000" is undefined
CSS transform warning: variable "--icon-link-fff" is undefined
CSS transform warning: variable "--icon-music-000" is undefined
CSS transform warning: variable "--icon-music-fff" is undefined
CSS transform warning: variable "--icon-picture-000" is undefined
CSS transform warning: variable "--icon-picture-fff" is undefined
CSS transform warning: variable "--icon-desktop-000" is undefined
CSS transform warning: variable "--icon-desktop-fff" is undefined
CSS transform warning: variable "--icon-phone-000" is undefined
CSS transform warning: variable "--icon-phone-fff" is undefined
CSS transform warning: variable "--icon-tablet-000" is undefined
CSS transform warning: variable "--icon-tablet-fff" is undefined
CSS transform warning: variable "--icon-user-000" is undefined
CSS transform warning: variable "--icon-checkmark-000" is undefined
CSS transform warning: variable "--icon-close-000" is undefined
CSS transform warning: variable "--icon-bundles-000" is undefined
CSS transform warning: variable "--icon-download-000" is undefined
CSS transform warning: variable "--icon-files-000" is undefined
CSS transform warning: variable "--icon-social-000" is undefined
CSS transform warning: variable "--icon-office-000" is undefined
CSS transform warning: variable "--icon-auth-000" is undefined
CSS transform warning: variable "--icon-monitoring-000" is undefined
CSS transform warning: variable "--icon-multimedia-000" is undefined
CSS transform warning: variable "--icon-organization-000" is undefined
CSS transform warning: variable "--icon-customization-000" is undefined
CSS transform warning: variable "--icon-integration-000" is undefined
CSS transform warning: variable "--icon-settings-dark-000" is undefined
CSS transform warning: variable "--icon-games-000" is undefined
CSS transform warning: variable "--icon-password-000" is undefined
CSS transform warning: variable "--icon-search-000" is undefined
CSS transform warning: variable "--icon-triangle-s-000" is undefined
CSS transform warning: variable "--icon-settings-dark-000" is undefined
CSS transform warning: variable "--color-main-background-darker" is undefined
CSS transform warning: variable "color-text-light" is undefined
CSS transform warning: variable "--icon-folder-000" is undefined
CSS transform warning: variable "--icon-recent-000" is undefined
CSS transform warning: variable "--icon-star-dark-000" is undefined
CSS transform warning: variable "--icon-share-000" is undefined
CSS transform warning: variable "--icon-public-000" is undefined
CSS transform warning: variable "--icon-external-000" is undefined
CSS transform warning: variable "--icon-delete-000" is undefined
CSS transform warning: variable "--icon-unshare-000" is undefined
CSS transform warning: variable "--icon-star-dark-FC0" is undefined
CSS transform warning: variable "--icon-tag-000" is undefined
New search handler registered
querying Whats New data was successful: nocontent
undefined
SEC7127: Umleitung wurde blockiert für CORS-Anforderung.
about:blank
```

Profile Page:
```
DOM7011: Der Code auf dieser Seite hat die Zwischenspeicherung für das Vor- und Zurücknavigieren deaktiviert. Weitere Informationen dazu finden Sie unter: http://go.microsoft.com/fwlink/?LinkID=291337
files
HTML1300: Navigation wurde ausgeführt.
user
DOM7011: Der Code auf dieser Seite hat die Zwischenspeicherung für das Vor- und Zurücknavigieren deaktiviert. Weitere Informationen dazu finden Sie unter: http://go.microsoft.com/fwlink/?LinkID=291337
files
HTML1500: Die Markierung kann nicht selbstschließend sein. Verwenden Sie eine explizite schließende Markierung.
user (94,21)
HTML1500: Die Markierung kann nicht selbstschließend sein. Verwenden Sie eine explizite schließende Markierung.
user (106,21)
HTML1500: Die Markierung kann nicht selbstschließend sein. Verwenden Sie eine explizite schließende Markierung.
user (118,21)
HTML1500: Die Markierung kann nicht selbstschließend sein. Verwenden Sie eine explizite schließende Markierung.
user (130,21)
HTML1508: Endmarkierung ohne Entsprechung.
user (214,50)
JQMIGRATE: Migrate is installed, version 1.4.0
SCRIPT1002: Syntaxfehler
notifications.js (7,98096)
SCRIPT1002: Syntaxfehler
settings.js (58,78)
CSS transform warning: variable "--color-text-details" is undefined
CSS transform warning: variable "--icon-confirm-fade-000" is undefined
CSS transform warning: variable "--color-checkbox-radio-border" is undefined
CSS transform warning: variable "--color-checkbox-radio-border" is undefined
CSS transform warning: variable "--color-checkbox-radio-disabled" is undefined
CSS transform warning: variable "--color-checkbox-radio-disabled" is undefined
CSS transform warning: variable "--icon-add-000" is undefined
CSS transform warning: variable "--icon-add-fff" is undefined
CSS transform warning: variable "--icon-address-000" is undefined
CSS transform warning: variable "--icon-address-fff" is undefined
CSS transform warning: variable "--icon-audio-000" is undefined
CSS transform warning: variable "--icon-audio-fff" is undefined
CSS transform warning: variable "--icon-audio-off-000" is undefined
CSS transform warning: variable "--icon-audio-off-fff" is undefined
CSS transform warning: variable "--icon-caret-fff" is undefined
CSS transform warning: variable "--icon-caret-000" is undefined
CSS transform warning: variable "--icon-checkmark-000" is undefined
CSS transform warning: variable "--icon-checkmark-fff" is undefined
CSS transform warning: variable "--icon-checkmark-46ba61" is undefined
CSS transform warning: variable "--icon-clippy-000" is undefined
CSS transform warning: variable "--icon-clippy-fff" is undefined
CSS transform warning: variable "--icon-close-000" is undefined
CSS transform warning: variable "--icon-close-fff" is undefined
CSS transform warning: variable "--icon-comment-000" is undefined
CSS transform warning: variable "--icon-comment-fff" is undefined
CSS transform warning: variable "--icon-confirm-000" is undefined
CSS transform warning: variable "--icon-confirm-fff" is undefined
CSS transform warning: variable "--icon-download-000" is undefined
CSS transform warning: variable "--icon-download-fff" is undefined
CSS transform warning: variable "--icon-confirm-fade-000" is undefined
CSS transform warning: variable "--icon-delete-000" is undefined
CSS transform warning: variable "--icon-delete-000" is undefined
CSS transform warning: variable "--icon-delete-e9322d" is undefined
CSS transform warning: variable "--icon-delete-fff" is undefined
CSS transform warning: variable "--icon-delete-fff" is undefined
CSS transform warning: variable "--icon-delete-fff" is undefined
CSS transform warning: variable "--icon-delete-e9322d" is undefined
CSS transform warning: variable "--icon-details-000" is undefined
CSS transform warning: variable "--icon-details-fff" is undefined
CSS transform warning: variable "--icon-edit-000" is undefined
CSS transform warning: variable "--icon-edit-fff" is undefined
CSS transform warning: variable "--icon-error-000" is undefined
CSS transform warning: variable "--icon-error-fff" is undefined
CSS transform warning: variable "--icon-error-e9322d" is undefined
CSS transform warning: variable "--icon-external-000" is undefined
CSS transform warning: variable "--icon-external-fff" is undefined
CSS transform warning: variable "--icon-fullscreen-000" is undefined
CSS transform warning: variable "--icon-fullscreen-fff" is undefined
CSS transform warning: variable "--icon-history-000" is undefined
CSS transform warning: variable "--icon-history-fff" is undefined
CSS transform warning: variable "--icon-info-000" is undefined
CSS transform warning: variable "--icon-info-fff" is undefined
CSS transform warning: variable "--icon-logout-000" is undefined
CSS transform warning: variable "--icon-logout-fff" is undefined
CSS transform warning: variable "--icon-mail-000" is undefined
CSS transform warning: variable "--icon-mail-fff" is undefined
CSS transform warning: variable "--icon-menu-000" is undefined
CSS transform warning: variable "--icon-menu-fff" is undefined
CSS transform warning: variable "--icon-more-000" is undefined
CSS transform warning: variable "--icon-more-fff" is undefined
CSS transform warning: variable "--icon-password-000" is undefined
CSS transform warning: variable "--icon-password-fff" is undefined
CSS transform warning: variable "--icon-pause-000" is undefined
CSS transform warning: variable "--icon-pause-fff" is undefined
CSS transform warning: variable "--icon-play-000" is undefined
CSS transform warning: variable "--icon-play-fff" is undefined
CSS transform warning: variable "--icon-play-add-000" is undefined
CSS transform warning: variable "--icon-play-add-fff" is undefined
CSS transform warning: variable "--icon-play-next-000" is undefined
CSS transform warning: variable "--icon-play-next-fff" is undefined
CSS transform warning: variable "--icon-play-previous-000" is undefined
CSS transform warning: variable "--icon-play-previous-fff" is undefined
CSS transform warning: variable "--icon-public-000" is undefined
CSS transform warning: variable "--icon-public-fff" is undefined
CSS transform warning: variable "--icon-quota-000" is undefined
CSS transform warning: variable "--icon-quota-fff" is undefined
CSS transform warning: variable "--icon-rename-000" is undefined
CSS transform warning: variable "--icon-rename-fff" is undefined
CSS transform warning: variable "--icon-screen-000" is undefined
CSS transform warning: variable "--icon-screen-fff" is undefined
CSS transform warning: variable "--icon-screen-off-000" is undefined
CSS transform warning: variable "--icon-screen-off-fff" is undefined
CSS transform warning: variable "--icon-search-000" is undefined
CSS transform warning: variable "--icon-search-fff" is undefined
CSS transform warning: variable "--icon-settings-000" is undefined
CSS transform warning: variable "--icon-settings-dark-000" is undefined
CSS transform warning: variable "--icon-settings-dark-fff" is undefined
CSS transform warning: variable "--icon-share-000" is undefined
CSS transform warning: variable "--icon-share-fff" is undefined
CSS transform warning: variable "--icon-share-fff" is undefined
CSS transform warning: variable "--icon-sound-000" is undefined
CSS transform warning: variable "--icon-sound-fff" is undefined
CSS transform warning: variable "--icon-sound-off-000" is undefined
CSS transform warning: variable "--icon-sound-off-fff" is undefined
CSS transform warning: variable "--icon-star-dark-000" is undefined
CSS transform warning: variable "--icon-star-000" is undefined
CSS transform warning: variable "--icon-star-fff" is undefined
CSS transform warning: variable "--icon-star-000" is undefined
CSS transform warning: variable "--icon-star-dark-FC0" is undefined
CSS transform warning: variable "--icon-star-000" is undefined
CSS transform warning: variable "--icon-star-dark-FC0" is undefined
CSS transform warning: variable "--icon-tag-000" is undefined
CSS transform warning: variable "--icon-tag-fff" is undefined
CSS transform warning: variable "--icon-timezone-000" is undefined
CSS transform warning: variable "--icon-timezone-fff" is undefined
CSS transform warning: variable "--icon-toggle-000" is undefined
CSS transform warning: variable "--icon-toggle-fff" is undefined
CSS transform warning: variable "--icon-toggle-background-000" is undefined
CSS transform warning: variable "--icon-toggle-background-fff" is undefined
CSS transform warning: variable "--icon-toggle-pictures-000" is undefined
CSS transform warning: variable "--icon-toggle-pictures-fff" is undefined
CSS transform warning: variable "--icon-triangle-e-000" is undefined
CSS transform warning: variable "--icon-triangle-e-fff" is undefined
CSS transform warning: variable "--icon-triangle-n-000" is undefined
CSS transform warning: variable "--icon-triangle-n-fff" is undefined
CSS transform warning: variable "--icon-triangle-s-000" is undefined
CSS transform warning: variable "--icon-triangle-s-fff" is undefined
CSS transform warning: variable "--icon-upload-000" is undefined
CSS transform warning: variable "--icon-upload-fff" is undefined
CSS transform warning: variable "--icon-user-000" is undefined
CSS transform warning: variable "--icon-user-fff" is undefined
CSS transform warning: variable "--icon-group-000" is undefined
CSS transform warning: variable "--icon-group-fff" is undefined
CSS transform warning: variable "--icon-video-000" is undefined
CSS transform warning: variable "--icon-video-fff" is undefined
CSS transform warning: variable "--icon-video-off-000" is undefined
CSS transform warning: variable "--icon-video-off-fff" is undefined
CSS transform warning: variable "--icon-video-switch-000" is undefined
CSS transform warning: variable "--icon-video-switch-fff" is undefined
CSS transform warning: variable "--icon-view-close-000" is undefined
CSS transform warning: variable "--icon-view-close-fff" is undefined
CSS transform warning: variable "--icon-view-download-000" is undefined
CSS transform warning: variable "--icon-view-download-fff" is undefined
CSS transform warning: variable "--icon-arrow-right-000" is undefined
CSS transform warning: variable "--icon-arrow-right-fff" is undefined
CSS transform warning: variable "--icon-view-pause-000" is undefined
CSS transform warning: variable "--icon-view-pause-fff" is undefined
CSS transform warning: variable "--icon-view-play-000" is undefined
CSS transform warning: variable "--icon-view-play-fff" is undefined
CSS transform warning: variable "--icon-arrow-left-000" is undefined
CSS transform warning: variable "--icon-arrow-left-fff" is undefined
CSS transform warning: variable "--icon-disabled-user-000" is undefined
CSS transform warning: variable "--icon-disabled-user-fff" is undefined
CSS transform warning: variable "--icon-disabled-users-000" is undefined
CSS transform warning: variable "--icon-disabled-users-fff" is undefined
CSS transform warning: variable "--icon-user-admin-000" is undefined
CSS transform warning: variable "--icon-user-admin-fff" is undefined
CSS transform warning: variable "--icon-calendar-fff" is undefined
CSS transform warning: variable "--icon-calendar-000" is undefined
CSS transform warning: variable "--icon-contacts-fff" is undefined
CSS transform warning: variable "--icon-contacts-000" is undefined
CSS transform warning: variable "--icon-files-fff" is undefined
CSS transform warning: variable "--icon-files-000" is undefined
CSS transform warning: variable "--icon-text-000" is undefined
CSS transform warning: variable "--icon-folder-000" is undefined
CSS transform warning: variable "--icon-folder-fff" is undefined
CSS transform warning: variable "--icon-folder-000" is undefined
CSS transform warning: variable "--icon-folder-drag-accept-000" is undefined
CSS transform warning: variable "--icon-home-000" is undefined
CSS transform warning: variable "--icon-home-fff" is undefined
CSS transform warning: variable "--icon-link-000" is undefined
CSS transform warning: variable "--icon-link-fff" is undefined
CSS transform warning: variable "--icon-music-000" is undefined
CSS transform warning: variable "--icon-music-fff" is undefined
CSS transform warning: variable "--icon-picture-000" is undefined
CSS transform warning: variable "--icon-picture-fff" is undefined
CSS transform warning: variable "--icon-desktop-000" is undefined
CSS transform warning: variable "--icon-desktop-fff" is undefined
CSS transform warning: variable "--icon-phone-000" is undefined
CSS transform warning: variable "--icon-phone-fff" is undefined
CSS transform warning: variable "--icon-tablet-000" is undefined
CSS transform warning: variable "--icon-tablet-fff" is undefined
CSS transform warning: variable "--icon-user-000" is undefined
CSS transform warning: variable "--icon-checkmark-000" is undefined
CSS transform warning: variable "--icon-close-000" is undefined
CSS transform warning: variable "--icon-bundles-000" is undefined
CSS transform warning: variable "--icon-download-000" is undefined
CSS transform warning: variable "--icon-files-000" is undefined
CSS transform warning: variable "--icon-social-000" is undefined
CSS transform warning: variable "--icon-office-000" is undefined
CSS transform warning: variable "--icon-auth-000" is undefined
CSS transform warning: variable "--icon-monitoring-000" is undefined
CSS transform warning: variable "--icon-multimedia-000" is undefined
CSS transform warning: variable "--icon-organization-000" is undefined
CSS transform warning: variable "--icon-customization-000" is undefined
CSS transform warning: variable "--icon-integration-000" is undefined
CSS transform warning: variable "--icon-settings-dark-000" is undefined
CSS transform warning: variable "--icon-games-000" is undefined
CSS transform warning: variable "--icon-password-000" is undefined
CSS transform warning: variable "--icon-search-000" is undefined
CSS transform warning: variable "--icon-triangle-s-000" is undefined
CSS transform warning: variable "--icon-settings-dark-000" is undefined
CSS transform warning: variable "--color-main-background-darker" is undefined
CSS transform warning: variable "color-text-light" is undefined
CSS transform warning: variable "--icon-personal-000" is undefined
CSS transform warning: variable "--icon-toggle-filelist-000" is undefined
CSS transform warning: variable "--icon-change-000" is undefined
CSS transform warning: variable "--icon-share-000" is undefined
CSS transform warning: variable "--icon-password-000" is undefined
CSS transform warning: variable "--color-text" is undefined
SEC7127: Umleitung wurde blockiert für CORS-Anforderung.
about:blank
```

**Comment by bobblkabb** [comment]
Confirmation.
Same here with internet explorer 11. Three dots menus are missing.

**Comment by danxuliu** [comment]
Fixed in #11291

Note, however, that if you apply the fix in that pull request (which will be included in Nextcloud 14.0.1) and log in again you may still not see the icons; I do not know if there is a way to manually clear the icon cache, @skjnldsv ?

**Comment by skjnldsv** [comment]
@danxuliu occ maintenance:repair or on every update :)

**Comment by mathiasconradt** [comment]
Got a report from a user:

> We have tested NC in Edge, Chrome, and Firefox.  It appears that the share icons do not appear in IE.  Do you have any suggestions?
> Here is an example.
 
![image](https://user-images.githubusercontent.com/295122/46022871-b7e2bf80-c0e3-11e8-804a-a22b3c656fcd.png)

**Comment by skjnldsv** [comment]
@mathiasconradt fixed on 14.0.1

**Comment by mathiasconradt** [comment]
@skjnldsv yes, confirmed, fixed in 14.0.1 - above mentioned user upgraded to 14.0.1 just now and it's gone there.
